src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+import core.basesyntax.supplier.FigureSupplier;
+import java.util.Random;
+
+public class Main {
+    public static final int ARRAY_SIZE = 6;
+    private static Random rnd = new Random();
+    private static Figure[] figures = new Figure[ARRAY_SIZE];
```

Comment:
`static` means that this field belongs to the class, in most cases we don't need static fields, except for constants
But overall this kind of variables feels like local vars, and services like `FigureSupplier` could be a class-level vars instead. 
So let's swap them.